kousa/lib/business-logic/room_logic.ex

```diff
@@ -189,10 +189,12 @@ defmodule Kousa.BL.Room do
     })
   end
 
+  # @deprecated
   def rename_room(_user_id, new_name) when byte_size(new_name) > 255 do
     {:error, "name needs to be less than 255 characters"}
   end
 
+  # @deprecated
   def rename_room(user_id, new_name) do
```

Comment:
handlers are a public interface so they need to be deprecated but all other elixir functions can be safely edited/deleted